python/semantic_kernel/connectors/ai/anthropic/services/anthropic_chat_completion.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import logging
+from collections.abc import AsyncGenerator
+from typing import Any
+
+from anthropic import AsyncAnthropic
+from anthropic.types import (
+    ContentBlockStopEvent,
+    Message,
+    RawContentBlockDeltaEvent,
+    RawMessageDeltaEvent,
+    RawMessageStartEvent,
+    TextBlock,
+)
+from pydantic import ValidationError
+
+from semantic_kernel.connectors.ai.anthropic.prompt_execution_settings.anthropic_prompt_execution_settings import (
+    AnthropicChatPromptExecutionSettings,
+)
+from semantic_kernel.connectors.ai.anthropic.settings.anthropic_settings import AnthropicSettings
+from semantic_kernel.connectors.ai.chat_completion_client_base import ChatCompletionClientBase
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+from semantic_kernel.contents.chat_history import ChatHistory
+from semantic_kernel.contents.chat_message_content import ITEM_TYPES, ChatMessageContent
+from semantic_kernel.contents.streaming_chat_message_content import ITEM_TYPES as STREAMING_ITEM_TYPES
+from semantic_kernel.contents.streaming_chat_message_content import StreamingChatMessageContent
+from semantic_kernel.contents.streaming_text_content import StreamingTextContent
+from semantic_kernel.contents.text_content import TextContent
+from semantic_kernel.contents.utils.author_role import AuthorRole
+from semantic_kernel.exceptions.service_exceptions import (
+    ServiceInitializationError,
+    ServiceResponseException,
+)
+from semantic_kernel.utils.experimental_decorator import experimental_class
+
+from .utils import ANTHROPIC_TO_SEMANTIC_KERNEL_FINISH_REASON_MAP, AnthropicFinishReason
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+@experimental_class
+class AnthropicChatCompletion(ChatCompletionClientBase):
+    """Antropic ChatCompletion class."""
+
+    async_client: AsyncAnthropic
+    
+    def __init__(
+        self,
+        ai_model_id: str | None = None,
+        service_id: str | None = None,
+        api_key: str | None = None,
+        async_client: AsyncAnthropic | None = None,
+        env_file_path: str | None = None,
+        env_file_encoding: str | None = None,
+    ) -> None:
+        """Initialize an AnthropicChatCompletion service.
+
+        Args:
+            ai_model_id (str): Anthropic model name, see
```

Comment:
I'm working on docs and one of the changes I'm making throughout is to remove the types from the docstring, they are read from the signature instead, so please remove them everywhere!